<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimesliceEditorDialog</class>
 <widget class="QDialog" name="TimesliceEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timeslice Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>4</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblw_timeslices">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Applied to Redmine?</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Total Duration:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbl_total_duration">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Current Duration:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbl_duration">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Update</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Start Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateTimeEdit" name="dte_update_start">
          <property name="displayFormat">
           <string>M/d/yyyy h:mm:ss AP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>End Date:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateTimeEdit" name="dte_update_end">
          <property name="displayFormat">
           <string>M/d/yyyy h:mm:ss AP</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_apply">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Apply</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/ui/res/edit.png</normaloff>:/ui/res/edit.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>New</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>End Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Start Date:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateTimeEdit" name="dte_add_start">
          <property name="displayFormat">
           <string>M/d/yyyy h:mm:ss AP</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_add_timeslice">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/ui/res/add.png</normaloff>:/ui/res/add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QDateTimeEdit" name="dte_add_end">
          <property name="displayFormat">
           <string>M/d/yyyy h:mm:ss AP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Delete</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Delete Selected Timeslice</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_delete_timeslice">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/ui/res/delete.png</normaloff>:/ui/res/delete.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
